src/step.cpp

```diff
@@ -108,7 +108,10 @@ void fields::phase_material() {
       }
     phasein_time--;
   }
-  if (or_to_all(changed)) {
+  am_now_working_on(MpiTime);
+  bool changed_mpi = or_to_all(changed);
+  finished_working();
+  if (changed_mpi) {
```

Comment:
It seems like all of this should be inside the `if (is_phasing()) { .... }` check.  That is, nothing should be executed in the common case of people who aren't using this feature.

Reply:
Fixed by ea21771.